puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/ci_centos_integration_test.groovy

```diff
@@ -0,0 +1,53 @@
+pipeline {
+    agent { label 'el' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Run simple ci.centos.org test') {
+            steps {
+                deleteDir()
+                git_clone_foreman_infra()
+                sh "mkdir jobs"
+
+                withCredentials([string(credentialsId: 'centos-jenkins', variable: 'PASSWORD')]) {
```

Comment:
Almost feels like we could introduce a runCentosCIPlaybook helper

Reply:
Yep -- I just didn't want to re-factor here given this was complex enough change.